canonical-web-and-design/microk8s.io
# Update instructions, position and github buttons

## Done

- updated the copy and move things around
    - github buttons
    - instructions

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8027/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- compare to the [copy doc](https://docs.google.com/document/d/1ObWmhRc4GCIhrBi8gT7yIIs0J_dn_8ravfLtcwhr3uQ/edit?usp=drivesdk)


## Issue / Card

Fixes #248

## Screenshots

![image](https://user-images.githubusercontent.com/441217/78254320-a6a31c00-74ed-11ea-873b-b9072218dce4.png)
